int32 target_color
float32[] center_B
float32[] center_R
float32[] center_Y
